nestkernel/event_delivery_manager.cpp

```diff
@@ -356,7 +383,21 @@ EventDeliveryManager::gather_spike_data_( const thread tid,
       // Needs to be called /after/ set_end_and_invalid_markers_.
       set_complete_marker_spike_data_( assigned_ranks, send_buffer_position, send_buffer );
 #pragma omp barrier
+      // TODO: Discuss, if the barrier above should be better placed after if-clause
```

Comment:
A decision should be made here and the todo should be removed.

Reply:
All threads should come to the same result for `gather_completed_checker_.all_true()`.
Hence, the barrier can stay inside the if-clause. Now removed comment.